packages/chart/src/FigureChartModel.ts

```diff
@@ -576,21 +656,21 @@ class FigureChartModel extends ChartModel {
     }
   }
 
-  getData() {
+  getData(): Partial<PlotData>[] {
     return this.data;
   }
 
-  getLayout() {
+  getLayout(): Partial<Layout> {
     return this.layout;
   }
 
-  getFilterColumnMap() {
+  getFilterColumnMap(): FilterColumnMap {
     return new Map(this.filterColumnMap);
   }
 
-  isFilterRequired() {
+  isFilterRequired(): boolean {
     return (
-      this.oneClicks.find(oneClick => oneClick.requireAllFiltersToDisplay) !=
+      this.oneClicks.find(oneClick => oneClick.isRequireAllFiltersToDisplay) !=
```

Comment:
Should not have the `is` prefix, that gets removed by the `@JsProperty` in the API code